utils/build/docker/nodejs/express4.Dockerfile

```diff
@@ -34,6 +35,3 @@ CMD ./app.sh
 COPY utils/build/docker/nodejs/install_ddtrace.sh binaries* /binaries/
 RUN /binaries/install_ddtrace.sh
 ENV DD_TRACE_HEADER_TAGS=user-agent
-
-# docker build -f utils/build/docker/nodejs.datadog.Dockerfile -t test .
-# docker run -ti -p 7777:7777 test
```

Comment:
hey, why remove that little reminder on how to use docker lol, someone put it there probably because it was useful to them no ?

Reply:
I did remove it because there is no `nodejs.datadog.Dockerfile` but If you think that it's needed as a little reminder I can put it back